syftlib/src/main/java/org/openmined/syft/processes/JobStatusSubscriber.kt

```diff
@@ -0,0 +1,14 @@
+package org.openmined.syft.processes
+
+open class JobStatusSubscriber {
```

Comment:
I have the feeling that we are mixing a reactive approach with callbacks and creating what it seems to me unnecessary complexity.

Reply:
The reason for mixing this is we dont want the end user of our library to understand the entire reactive approach. Plus it goes with how the roadmap specifies things to be